plugins/Editing.jsx

```diff
@@ -233,10 +255,30 @@ class Editing extends React.Component {
             </SideBar>
         );
     }
+
+    checkVisibility = (selectedLayer, visibility) => {
+        if (selectedLayer != null){
+            let layer = this.props.layers.filter(layer => (layer.role === LayerRole.THEME && LayerUtils.searchSubLayer(layer, 'name', selectedLayer)))[0];
+            let path = this.findParents(layer, selectedLayer);
```

Comment:
Don't you want to use the path returned by LayerUtils.searchSubLayer?

Reply:
Actually you can also use this.props.layer.find instead of filter.

Reply:
thank you for your guidance